src/client/inkeep/useInkeepSettings.ts

```diff
@@ -0,0 +1,74 @@
+import type {
+  InkeepAIChatSettings,
+  InkeepSearchSettings,
+  InkeepBaseSettings,
+  InkeepModalSettings,
+} from '@inkeep/uikit';
+import { AIChatDisclaimerSettings } from '@inkeep/uikit';
+
+type InkeepSharedSettings = {
+  baseSettings: InkeepBaseSettings;
+  aiChatSettings: InkeepAIChatSettings;
+  searchSettings: InkeepSearchSettings;
+  modalSettings: InkeepModalSettings;
+};
+
+const useInkeepSettings = (): InkeepSharedSettings => {
+  const baseSettings: InkeepBaseSettings = {
+    apiKey: 'b0537306817fb8a0daea377df2b273d1b00ac709182d1dc7',
```

Comment:
Maybe it's better practice for these to be environment variables? 
https://docs.inkeep.com/integrations/nextjs/chat-button

Reply:
Yeah, good point. Just changed it.